### ISSUE TITLE
comprehension of 

### ISSUE THREAD

**Issue by as142** [MasterVictor/TiendaRopa#1]
Hi there, When I was learning and trying to understand the function "is_possible_short_number" in TiendaRopa/Lib/site-packages/phonenumbers/shortnumberinfo.py, I was confused about the description of its delivered function. My understanding of this code snippet is "check whether a number is either a number". Do I understand that right?

**Comment by MasterVictor** [comment]
yes, its function is to accept only phone numbers that can be real.
